storage/src/main/java/org/apache/uniffle/storage/handler/impl/LocalFileWriteHandler.java

```diff
@@ -51,19 +53,10 @@ public LocalFileWriteHandler(
   }
 
   private void createBasePath() {
-    File baseFolder = new File(basePath);
-    // check if shuffle folder exist
-    if (!baseFolder.exists()) {
-      try {
-        // try to create folder, it may be created by other Shuffle Server
-        baseFolder.mkdirs();
-      } catch (SecurityException e) {
-        // if folder exist, ignore the exception
-        if (!baseFolder.exists()) {
-          LOG.error("Can't create shuffle folder:" + basePath, e);
-          throw e;
-        }
-      }
+    try {
+      Files.createDirectories(Paths.get(basePath));
```

Comment:
> Creates a directory by creating all nonexistent parent directories first. Unlike the createDirectory method, an exception is not thrown if the directory could not be created because it already exists.

Isn't it exactly what we want?

Reply:
> Yeah. but there's q shortcut, if the dir is already existed, we don't have to invoke `createDirectories`, which seems quite complex compared to check the existence of dir.

If there will be a significant performance difference, why doesn't the library author do this in the `createDirectories`?
Even if it can improve the case when the dir already exists, it will worsen the case when the dir doesn't exist.

Reply:
We will have to check existence and throw exception when a file (not a directory) exists for each parent directory.
Which is complex and repeated work since it's already done in `Files.createDirectories()`.

Reply:
OK, I got it.